### ISSUE TITLE
Change default photos location

### ISSUE THREAD

**Issue by ksz16** [FossifyOrg/Camera#18]
### Checklist

- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Camera/issues) or [closed](https://github.com/FossifyOrg/Camera/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Camera/discussions) or [closed](https://github.com/FossifyOrg/Camera/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Camera/blob/master/CONTRIBUTING.md).
- [ ] I optionally [donated](https://fossify.org/donate) to support the Fossify mission.

### Feature description

In [Simple Camera](https://f-droid.org/packages/com.simplemobiletools.camera/) default location for photos is the DCIM folder. Could you please consider changing this path to DCIM/Camera?

### Why do you want this feature?

DCIM/Camera is more common in applications of this type.

**Comment by samsoninja** [comment]
Agree. DCIM/Camera is default for most camera apps. Conventionally, images are saved in a subdirectory of DCIM rather than directly in there, and Fossify camera should be no exception.
